packages/lit-dev-content/site/home/2-advantages.html

```diff
@@ -20,7 +20,7 @@ <h1>Simple</h1>
         <img fetchpriority="high" loading="lazy" src="{{site.baseurl}}/images/home/lightning.svg" width="300" height="300" alt="" />
         <h1>Fast</h1>
       </div>
-      <p>Tiny footprint, instant updates</p>
+      <p>Tiny potato, instant updates</p>
```

Comment:
I don't think this was intentional? As in 🥔 or 🍠❓

Reply:
Hahaha. I was testing out `--watch` mode. Reverted :grin: